docs/admin/installation/hardware.rst

```diff
@@ -449,8 +450,33 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
-.. _nucc12_recommendation:
-.. _intel-12th-gen-nuc:
```

Comment:
Why was this anchor removed? It was intentionally added in 5f098d78fb98ac8c9c3a021aad0f5730fe52204e.

Reply:
(As above)